### DISCUSSION TITLE
YOURLS installed in Subdomain

### DISCUSSION THREAD

**Discussion by master3395** [YOURLS/YOURLS#2869] [Q&A]
Hello, i am trying to have yourls installed in a sub domain, not domain.com/subdomain but subdomain.domain.com
And I get weird redirects and errors.

I tried the tutorial here: [https://github.com/YOURLS/YOURLS/wiki/.htaccess#case-yourls-installed-in-subdirectory](https://github.com/YOURLS/YOURLS/wiki/.htaccess#case-yourls-installed-in-subdirectory)
But I get the same result.

I have tried to change the role and permission for the whole folder.
chmod -R 777 /home/domain.com/yourls.domain.com/

I am running yourls on my own vps that has cyberpanel installed. (CyberPanel installs OpenLiteSpeed)
It is running Centos 8 with Webmin and Lamp installed.

Core Php is 7.4.14

The front end is fine, but the admin page shows the error below.

403
Forbidden
Access to this resource on the server is denied!

**Comment by ozh** [comment]
Actual .htaccess ? Actual config.php ? Actual URL of your setup ? There is not one useful info here.

Running YOURLS off domain.com or sub.dom.ain.com is exactly the same.

**Reply by master3395** [reply]
Here are the files.
[config.txt](https://github.com/YOURLS/YOURLS/files/6137876/config.txt)
[.htaccess.txt](https://github.com/YOURLS/YOURLS/files/6137877/default.htaccess.txt)

**Reply by dgw** [reply]
Those look configured for a sub-_directory_, not a subdomain. Am I missing something?

Like `https://newstargeted.com/yourls` for `YOURLS_SITE`.

**Comment by dgw** [comment]
@ozh Can we start directing troubleshooting questions like this to Discussions?
